PKGBUILD

```diff
@@ -4,15 +4,14 @@
 # Contributor: demostanis worlds <demostanis@protonmail.com>
 
 pkgname=archinstall
-pkgver=2.6.0
+pkgver=2.7.2
 pkgrel=1
 pkgdesc="Just another guided/automated Arch Linux installer with a twist"
 arch=(any)
 url="https://github.com/archlinux/archinstall"
-license=(GPL3)
+license=(GPL-3.0-or-later)
 depends=(
   'arch-install-scripts'
-  'btrfs-progs'
```

Comment:
Does archinstall no longer support btrfs or is support for the filesystem achieved in another way?

Reply:
humm, crap you're right. It needs it in the ISO to also be able to do `mkfs.btrfs`. Good catch! I was thinking it just needed it to be installed *inside* the system.